server/server_test.go

```diff
@@ -45,12 +45,14 @@ func TestOpenFGAServer(t *testing.T) {
 		uri := testDatastore.GetConnectionURI()
 		ds, err := postgres.NewPostgresDatastore(uri)
 		require.NoError(t, err)
+		defer ds.Close(context.Background())
```

Comment:
Slightly out of scope for this PR, but would you mind removing the context argument from the `Close` function. None of the implementations use it so I don't know what it is doing there anyway.